support-frontend/assets/pages/contributions-landing/components/ContributionFormFields.tsx

```diff
@@ -115,18 +121,20 @@ function ContributionFormFields(props: PropTypes) {
 					type="email"
 					autoComplete="email"
 					supporting="example@domain.com"
-					onInput={props.updateEmail}
+					onChange={(e) => updateEmail(e.target.value)}
```

Comment:
is there any functional impact from changing onInput -> onChange?

Reply:
Good point - I was going to highlight this one! So in `React`, `onChange` is the same as `onInput` (triggers on every keystroke), which is different to the default DOM `onchange` event (only triggers after exiting the field)